linux_os/guide/system/network/network-firewalld/ruleset_modifications/ensure_firewall_rules_for_open_ports/tests/wrong.fail.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Testing in a local VM with RHEL8, which also uses firewalld, this test scenario is reporting error instead of `fail`. Could you check and confirm it or the OVAL is correct, please? The errors seems to appear when comparing the port value to the expected state.

Reply:
Thanks for the feedback 🙇 [f498cc4](https://github.com/ComplianceAsCode/content/pull/9950/commits/f498cc45eb29fb9af681655d3c0a57bb63c3a275) and [3ed0c96](https://github.com/ComplianceAsCode/content/pull/9950/commits/3ed0c967194a808b52de4735d94c48cbc94ed395) should address that issue

Reply:
Unfortunately, the `ssg-test_listening_fw_inet_service_ports_tcp_test` is still causing error in the general assessment:

![Screenshot from 2023-03-29 12-08-31](https://user-images.githubusercontent.com/2074099/228584103-7074a68a-a195-4c91-95d1-0af0ab32e50a.png)

Reply:
Could not see that issue in the reports after last commits

Reply:
Yes, it is no longer happening. Thanks.
I have only one additional comment here. Could you include the `# remediation = none` header, please? This is to avoid warning messages in `fail` test scenario scripts.